src/algorand/AlgorandEngine.ts

```diff
@@ -729,6 +729,11 @@ export class AlgorandEngine extends CurrencyEngine<
       edgeTransaction.txid = txId
       edgeTransaction.date = Date.now() / 1000
     } catch (e: any) {
+      // Recipient did not activate what we're trying to send
```

Comment:
Relying on a string would work but it's a fragile solution. Instead, let's query for the address's information and see if the asset is enabled. You can use the fetchAccountInfo helper and move the check into signTx